### PR TITLE
Update translations to use #format

### PR DESCRIPTION
**Pull request by Mikusch**

### CODE REVIEW

File: addons/sourcemod/scripting/tfgo.sp [hunk 1]
```diff
@@ -608,7 +608,7 @@ void PlantBomb(TFTeam team, int cpIndex, ArrayList cappers)
 	
 	// Show text on screen
 	char message[PLATFORM_MAX_PATH];
-	Format(message, sizeof(message), "%T", "Bomb_Planted", LANG_SERVER, tfgo_bombtimer.IntValue);
+	Format(message, sizeof(message), "%t", "Bomb_Planted", tfgo_bombtimer.IntValue);
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/scripting/tfgo.sp:611]: `Format` don't know which client language to translate as, best to keep `%T`

File: addons/sourcemod/scripting/tfgo.sp [hunk 2]
```diff
@@ -539,7 +539,7 @@ void PlantBomb(TFTeam team, int cpIndex, ArrayList cappers)
 	for (int i = 0; i < cappers.Length; i++)
 	{
 		int capper = cappers.Get(i);
-		TFGOPlayer(capper).AddToAccount(tfgo_cash_player_bomb_planted.IntValue, "%T", "Player_Cash_Award_Bomb_Planted", LANG_SERVER, tfgo_cash_player_bomb_planted.IntValue);
+		TFGOPlayer(capper).AddToAccount(tfgo_cash_player_bomb_planted.IntValue, "%t", "Player_Cash_Award_Bomb_Planted", tfgo_cash_player_bomb_planted.IntValue);
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/scripting/tfgo.sp:542]: `TFGOPlayer.AddToAccount` Should use `SetGlobalTransTarget` to set language as target `this`

File: addons/sourcemod/scripting/tfgo/buymenu.sp [hunk 3]
```diff
@@ -6,7 +6,7 @@
 bool BuyMenu_DisplayMainBuyMenu(int client)
 {
 	Menu menu = new Menu(MenuHandler_MainBuyMenu, MenuAction_Display | MenuAction_Select | MenuAction_Cancel | MenuAction_End | MenuAction_DisplayItem);
-	menu.SetTitle("%T\n%T", "BuyMenu_Title", LANG_SERVER, "BuyMenu_SelectSlot", LANG_SERVER);
+	menu.SetTitle("%t\n%t", "BuyMenu_Title", "BuyMenu_SelectSlot");
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/scripting/tfgo/buymenu.sp:9]: `Menu.SetTitle` also don't know which client to get language

File: addons/sourcemod/scripting/tfgo/event.sp [hunk 4]
```diff
@@ -174,8 +173,8 @@ Action Event_ArenaWinPanel(Event event, const char[] name, bool dontBroadcast)
 	{
 		TFGOTeam red = TFGOTeam(TFTeam_Red);
 		TFGOTeam blue = TFGOTeam(TFTeam_Blue);
-		red.PrintToChat("%T", "Team_Cash_Award_no_income_stalemate", LANG_SERVER);
-		blue.PrintToChat("%T", "Team_Cash_Award_no_income_stalemate", LANG_SERVER);
+		red.PrintToChat("%t", "Team_Cash_Award_no_income_stalemate");
+		blue.PrintToChat("%t", "Team_Cash_Award_no_income_stalemate");
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/scripting/tfgo/event.sp:177]: `TFGOTeam.PrintToChat` and `TFGOTeam.AddToClientAccounts` Should format string separately to each clients to print different languages, similar to how `PrintToChatAll` stock is done

File: addons/sourcemod/translations/tfgo.phrases.txt [hunk 5]
```diff
@@ -221,69 +223,85 @@
...
 	{
-		"en"	"{positive}+$%d{default}: Team award for planting the bomb."
+		"#format"	"{1:d}"
+		"en"		"{positive}+${1}{default}: Team award for planting the bomb."
 	}
 	"Team_Cash_Award_no_income_stalemate"
 	{
-		"en"	"{negative}0{default}: No income for triggering a stalemate."
+		"#format"	"{1:d}"
+		"en"		"{negative}0{default}: No income for triggering a stalemate."
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/translations/tfgo.phrases.txt:304]: `#format` looks unneeded here

**Comment by Mikusch** [comment]
@Blueberryy I assume you wish to update russian translations to this format? If not, I can do this myself in a later commit. Merging this for now.

**Pull request merged**